metadata-editor/app/lib/Syndication.scala

```diff
@@ -1,67 +1,82 @@
 package lib
 
 import com.gu.mediaservice.lib.aws.{DynamoDB, NoItemFound, UpdateMessage}
+import com.gu.mediaservice.lib.logging.GridLogging
 import com.gu.mediaservice.model.{Edits, Photoshoot, SyndicationRights}
 import com.gu.mediaservice.syntax.MessageSubjects
-import play.api.libs.json.{JsNull, JsString}
+import org.joda.time.DateTime
+import play.api.libs.json.{JsNull, JsString, Reads}
 
 import scala.concurrent.{ExecutionContext, Future}
 
-trait Syndication extends Edit with MessageSubjects {
+trait Syndication extends Edit with MessageSubjects with GridLogging {
 
   def syndicationStore: SyndicationStore
 
   private val syndicationRightsFieldName = "syndicationRights"
 
-  def deletePhotoshootAndPublish(id: String)
-                                (implicit ec: ExecutionContext): Future[Unit.type] = {
+  private def timedFuture[T](label: String, f : => Future[T])(implicit ec: ExecutionContext): Future[T] = {
+    val timeBefore = DateTime.now().getMillis
+    val result = f
+      result.onComplete { _ =>
+        val timeAfter = DateTime.now().getMillis
+        logger.info(s"$label took ${timeAfter - timeBefore} milliseconds")
+      }
+    result
+  }
```

Comment:
This looks like a [stopwatch](https://github.com/guardian/grid/blob/main/common-lib/src/main/scala/com/gu/mediaservice/lib/logging/Stopwatch.scala), can we reuse it?